### PR TITLE
chore(docs): add migration to KaizenProvider

### PR DESCRIPTION
**Pull request by gyfchong**

### CODE REVIEW

File: packages/components/src/KaizenProvider/docs/KaizenProvider.stories.mdx [hunk 1]
```diff
@@ -1,11 +1,15 @@
+
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/docs/KaizenProvider.stories.mdx:1]: 👀

File: packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+-     services={[...defaultPreset, ThemeProvider]}
++     services={[...defaultPreset, KaizenProvider]}
+      options={{
+        ...
+-       themeManager,
+      }}
+  >
+   ...
+  </FrontendServices>
+)
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff:19]: New line EOF (other diff file too)? :)

File: packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
++ import { KaizenProvider } from "@kaizen/components"
+
+- const themeManager = new ThemeManager(defaultTheme)
+
+...
+
+return(
+  <FrontendServices
+-     services={[...defaultPreset, ThemeProvider]}
++     services={[...defaultPreset, KaizenProvider]}
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff:11]: Is `defaultPreset` meant to be `defaultTheme`?

File: packages/components/src/KaizenProvider/docs/code-diffs/migration-to-v1.diff [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
++ import { KaizenProvider, ThemeManager } from "@kaizen/components"
+
+const themeManager = new ThemeManager(defaultTheme)
+
+...
+
+return(
+  <FrontendServices
+-     services={[...defaultPreset, ThemeProvider]}
++     services={[...defaultPreset, KaizenProvider]}
```
Comment (HeartSquared) [inline packages/components/src/KaizenProvider/docs/code-diffs/migration-to-v1.diff:12]: Is `defaultPreset` meant to be `defaultTheme`?

File: packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff (hunk 3 above)
Comment (gyfchong) [reply packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff:11]: Nope, that's the FEF preset

Comment (HeartSquared) [reply packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff:11]: Oh. That's very confusing in this example. Maybe add a comment or put an example value for the variable?

Comment (gyfchong) [reply packages/components/src/KaizenProvider/docs/code-diffs/migration-bonus.diff:11]: Hm, that value comes from a separate package so I don't think I want to elude people to thinking they should be managing that value. I'll add a description to mention this example is from the FE-Template, hopefully less confusion.

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**